sandbox/src/main/java/org/robolectric/internal/bytecode/ShadowDecorator.java

```diff
@@ -22,7 +22,7 @@ public void decorate(MutableClass mutableClass) {
     mutableClass.addField(
         0,
         new FieldNode(
-            Opcodes.ACC_PUBLIC | Opcodes.ACC_SYNTHETIC,
+            Opcodes.ACC_PUBLIC | Opcodes.ACC_TRANSIENT,
```

Comment:
Can this also include synthetic? `Opcodes.ACC_PUBLIC | Opcodes.ACC_TRANSIENT | Opcodes.ACC_SYNTHETIC`

Synthetic is useful to convey that a field was added behind-the-scenes and is not derived from the source code of the class. It should be a no-op in terms of functionality.